topi/include/topi/nn.h

```diff
@@ -86,6 +115,7 @@ inline tvm::Tensor relu(const tvm::Tensor& t,
 inline tvm::Tensor pad(const tvm::Tensor& t,
                        const tvm::Array<tvm::Expr>& pad_before,
                        tvm::Array<tvm::Expr> pad_after = tvm::Array<tvm::Expr>(),
+                       Expr* pad_value = nullptr,
```

Comment:
We can simply allow pass in pad_value as ```Expr()``` and use pad_value.defined() to see if it is defined